nvmw.bat

```diff
@@ -75,27 +81,44 @@ exit /b 0
 :install
 setlocal
 
+set NODE_TYPE=node

```

Comment:
@hakobera should we keep `NODE_TYPE` be empty string when install node.js? This can keep compatible with the old way.